fossunited/chapters/doctype/foss_event_rsvp/foss_event_rsvp.json

```diff
@@ -120,13 +122,25 @@
       "fieldname": "event_name",
       "fieldtype": "Data",
       "label": "Event Name"
+    },
+    {
+      "default": "1",
+      "fieldname": "accept_all_incoming",
+      "fieldtype": "Check",
+      "label": "Accept All Incoming Submissions"
```

Comment:
could we find a better name?

i'm asking because thinking of `"Accept all incoming submissions"` set to False feels like "don't accept all incoming submissions" and not "submissions need approval".

could we maybe go with `rsvp_submissions_need_approval` or `submissions_need_approval` or something of that sort? the code reads better IMO

```
if rsvp.submissions_need_approval:
    # set submission status to pending
# set submission status to accepted
```

Reply:
Changing this to Submissions Need Approval